test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -248,20 +248,22 @@ TEST_F(ClusterManagerImplTest, MaxClusterName) {
                             "document key: #/name");
 }
 
-TEST_F(ClusterManagerImplTest, InvalidClusterNameChars) {
+TEST_F(ClusterManagerImplTest, ValidClusterName) {
   const std::string json = R"EOF(
   {
     "clusters": [
     {
-      "name": "cluster:"
+      "name": "cluster:name",
+      "connect_timeout_ms": 250,
+      "type": "static",
+      "lb_type": "round_robin",
+      "hosts": [{"url": "tcp://127.0.0.1:11001"}]
     }]
   }
   )EOF";
 
-  EXPECT_THROW_WITH_MESSAGE(create(parseBootstrapFromJson(json)), Json::Exception,
-                            "JSON at lines 4-6 does not conform to schema.\n Invalid schema: "
-                            "#/properties/name\n Schema violation: pattern\n Offending document "
-                            "key: #/name");
+  create(parseBootstrapFromJson(json));
+  EXPECT_EQ(0, factory_.stats_.counter("cluster.cluster_name.lb_local_cluster_not_ok").value());
```

Comment:
Are there any non-zero stats that I can check here?

Reply:
Can't you check any cluster stat that is per-cluster, adding a `:` in there?

Reply:
These are the cluster stats that I see: https://github.com/envoyproxy/envoy/blob/master/include/envoy/upstream/upstream.h#L177